src/api/sounds.ts

```diff
@@ -0,0 +1,38 @@
+import firebase from "firebase/app";
+
+import { SoundConfig, SoundConfigMap, SoundConfigSchema } from "types/sounds";
+
+import { withId } from "utils/id";
+import { itemsToObjectByIdReducer } from "utils/reducers";
+
+export const fetchSoundConfigs = async (): Promise<SoundConfigMap> => {
+  try {
+    const soundConfigsSnapshot = await firebase
+      .firestore()
+      .collection("sounds")
+      .withConverter(soundConfigConverter)
+      .get();
```

Comment:
[OOS, future] Currently this will load everything in the `sounds` collection, which IMO is fine for this MVP version of the feature.

Once we flesh it out/polish it a bit more (eg. adding admin tooling, etc) it would also make sense to change this part too, perhaps to load sounds from the 'world' object (once it exists), or similar?